src/gateway/hal/master_controller_classic.py

```diff
@@ -1164,12 +1164,35 @@ def get_backup(self):
                 time.sleep(2)  # Doing heavy reads on eeprom can exhaust the master. Give it a bit room to breathe.
         return ''.join(chr(c) for c in output)
 
-    def factory_reset(self):
-        # type: () -> None
+    def factory_reset(self, Can=False):
```

Comment:
Capitalized variables are not really idiomatic in python.

```suggestion
    def factory_reset(self, can=False):
```